src/features/search/index.js

```diff
@@ -14,14 +15,23 @@ export const Search = () => {
   const onInputChange = ({ target }) => {
     dispatch(changeQuery(target.value));
     if (target.value === "") return;
-    else if (navigationSelector === "movies") { dispatch(fetchSearchMoviesList(target.value.trim())) }
-    else dispatch(fetchSearchPeopleList(target.value.trim()));
+    else if (navigationSelector === "movies") dispatch(fetchSearchMoviesList({ query: target.value.trim(), page: 1 }))
+    else dispatch(fetchSearchPeopleList({ query: target.value.trim(), page: 1 }));
 
```

Comment:
line 17 and 18:  
here page is set to 1 so that it shows the first page after searching